test/tests/test_harness/diff_gold/csvdiff_diff_gold_out.csv

```diff
@@ -0,0 +1,7 @@
+time,num_dofs
```

Comment:
Git supports soft links. I'd recommend that you remove these copies and commit soft links (e.g. "ln -s <orig> <link>") instead.

Reply:
much simpler with the ln -s, thanks for the tip!